server/grpc_service.go

```diff
@@ -462,6 +462,41 @@ func (s *GrpcServer) AllocID(ctx context.Context, request *pdpb.AllocIDRequest)
 	}, nil
 }
 
+// RecoverAllocID implements gRPC PDServer.
+func (s *GrpcServer) RecoverAllocID(ctx context.Context, request *pdpb.RecoverAllocIDRequest) (*pdpb.RecoverAllocIDResponse, error) {
```

Comment:
Do we need both HTTP and gRPC interfaces?

Reply:
https://github.com/tikv/pd/pull/5181#discussion_r948666633